CONTRIBUTE.md

```diff
@@ -0,0 +1,17 @@
+The following is based off of the [Rails Contribution guidelines][].
+
+Spacemacs is a volunteer effort. We encourage you to pitch in. The community makes Spacemacs what it is. We have a few guidelines, which we ask all contributors to follow.
+
+- If you want to ask a usage question, be sure to read [our documentation][], as it may hold the answer. You should also reference the `README.org` of the relevant layer(s). If your question is not answered there, then feel free to ask on our [gitter chat][].
+- If you want to submit a PR, please read [our contribution guidelines][] before submitting. Also, make sure that your Pull Request (PR) is against the `develop` branch and, not `master`.
```

Comment:
The explanation for the abbreviation "PR" is on its second usage, not its first.